planner/core/physical_plans.go

```diff
@@ -404,6 +404,16 @@ type PhysicalHashAgg struct {
 	basePhysicalAgg
 }
 
+// NewPhysicalHashAgg creates a new PhysicalHashAgg from a LogicalAggregation.
+func NewPhysicalHashAgg(la *LogicalAggregation, expectedCnt float64, prop *property.PhysicalProperty) *PhysicalHashAgg {
+	agg := basePhysicalAgg{
+		GroupByItems: la.GroupByItems,
+		AggFuncs:     la.AggFuncs,
+	}.initForHash(la.ctx, la.stats.ScaleByExpectCnt(expectedCnt), la.blockOffset, prop)
+	agg.SetSchema(la.schema.Clone())
```

Comment:
When this function is called in cascades planner, `la.stats` and `la.schema` is not set properly actually, because we only maintain schema and stats info at `Group` level?

Reply:
Yes, addressed.